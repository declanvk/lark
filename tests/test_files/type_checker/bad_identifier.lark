def new(msg: bool,) -> bool {
    msg1
    //~ ERROR: unknown identifier `msg1`
}