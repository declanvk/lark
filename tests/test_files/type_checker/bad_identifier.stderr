error: unknown identifier `msg1`
- type_checker/bad_identifier:2:4
2 |     msg1
  |     ^^^^
